packages/core/echo/echo-db/src/database.ts

```diff
@@ -98,6 +98,10 @@ export class EchoDatabaseImpl implements EchoDatabase {
     return this._automerge.spaceKey;
   }
 
+  async open() {}
```

Comment:
why have these if they aren't implemented? comment to explain?

Reply:
Stubs for the future use, just so we can structure our code around them being there. --- added a comment

Reply:
see comment about open/close context/resources/interface aspect.